src/components/core/__stories__/Fusion.stories.tsx

```diff
@@ -3,9 +3,32 @@ import { storiesOf } from '@storybook/react';
 import FusionHeader from '../Header';
 import FusionContent from '../Content';
 
+const AppSelectorIcon = () => {

```

Comment:
Could this be moved to Icons dir?

Reply:
The original thought was to keep the app selector icon in the fusion portal since no other apps or anything should display the app selector. In theory it could be moved to the icons dir, but that would open up for developers to use the icon for other purposes